packages/psss-server/src/app/addRoutesToApp.ts

```diff
@@ -74,6 +75,7 @@ export function addRoutesToApp(app: Express) {
    * PUT routes
    */
   app.put('/v1/weeklyReport/:countryCode/:siteCode?', handleWith(SaveWeeklyReportRoute));
+  app.put('/v1/alerts/:alertId/archive', handleWith(ArchiveAlertRoute));
```

Comment:
Potentially we can change this endpoint to `/v1/alerts/:alertId/:state`, and the Route class can handle changing the state to outbreak/archive/activeAlert